### ISSUE TITLE
0.9.0-rc3: Credo Disable Checks Seemingly Not Honored In --format=one-line

### ISSUE THREAD

**Issue by asummers** [rrrene/credo#504]
### Environment

* Credo version (`mix credo -v`): `0.9.0-rc3`
* Erlang/Elixir version (`elixir -v`):
```
Erlang/OTP 20 [erts-9.1.1] [source] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]

Elixir 1.6.1 (compiled with OTP 20)
```
* Operating system: OSX

### What were you trying to do?
Run `mix credo --format=one-line --strict`

### Expected outcome
`# credo disable-XXX` lines are honored

### Actual outcome
`# credo disable-XXX` lines are not honored and errors that are disabled are showing up with warnings. Strangely enough, if you remove the `--format=one-line` part, Credo reports all checks passing. 

```
Checking 1123 source files ...

Please report incorrect results: https://github.com/rrrene/credo/issues

Analysis took 8.0 seconds (1.3s to load, 6.6s running checks)
5925 mods/funs, found no issues.
```

This worked in at least `0.8.10`, which is the version I bumped from.

**Comment by asummers** [comment]
Noticed this is a dupe.
